tantivy/__init__.py

```diff
@@ -1 +1 @@
-from .tantivy import *
+from .tantivy import *
```

Comment:
Is there a reason why we are removing these?

Many unix tools expect them to be there, e.g:
```bash
$ wc -l tantivy/__init__.py 
0 tantivy/__init__.py
```

Reply:
no reason I am just making a mistake. I did not know about `wc -l` not counting that as a line though. 
I like a trailing \n for `cat`